internal/testplanet/planet.go

```diff
@@ -403,6 +403,7 @@ func (planet *Planet) newSatellites(count int) ([]*satellite.Peer, error) {
 					Email:  prefix + "@example.com",
 					Wallet: "0x" + strings.Repeat("00", 20),
 				},
+				QueryTimeout: time.Duration(60) * time.Second, // TODO: why didn't it default to 60s when not set here?
```

Comment:
`60 * time.Second`, no need for the conversion.

All params in testplanet must be explicitly specified because it ignores all of the config tags.

Reply:
ah, good to know.